components/post-text/PostText.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Link from 'next/link';
+import { uniq as _uniq } from 'lodash';
+
+interface IProps {
+  text: string;
+}
+
+const PostText = ({ text }: IProps) => {
+  const regexExp = /#[\p{L}\p{M}0-9_]+/gu;
+
+  const getAndRenderHashTags = (text: string) => {
+    const matches = text.match(regexExp);
+
+    if (!matches) return null;
+    const uniqMatches = _uniq(matches);
```

Comment:
this might be an JS alternative to loadash's function `_uniq`:
`const uniqMatches = Array.from(new Set(matches));`
do we really want to use loadash? Isn't there almost always an JS alternative?